src/main.js

```diff
@@ -198,6 +198,23 @@ function userCreatedPoster() {
   showMainPoster();
 }
 
+function savePoster() {
+  var save = {
+    image: posterImage.src,
+    title: posterTitle.innerText,
+    quote: posterQuote.innerText,
+  }
+  for (var i = 0; i < savedPosters.length; i++){
+    if (savedPosters.includes(save.image && save.title && save.quote) === false) {
```

Comment:
Let's say `save.img = 'turtle'`, `save.title = 'turtle'` and `save.title = 'Your code rocks!'`; wouldn't `savedPosters.includes` be checking that doesn't look like `['turtle', 'turtle', 'Your code rocks!']`?

Reply:
Thanks Greyson! You rock, rock! Thanks for pointing us in the right direction.